worker/Cargo.toml

```
@@ -14,4 +14,5 @@ rand = "0.7.2"
 redis = "0.12.0"
 reqwest = "0.9.22"
 url = "^2.1.0"
-scraper = "^0.11.0"
\ No newline at end of file
+scraper = "^0.11.0"
+mockito = "0.21.0"
```

```suggestion
mockito = "^0.21.0"
```

We might want to allow for automatically updating this tool.